### PR TITLE
[KIECLOUD-663] Create pipeline for rhpam-kogito operator

### PR DESCRIPTION
**Pull request by desmax74**
WIP 
See: https://issues.redhat.com/browse/KIECLOUD-663
Signed-off-by: desmax74 <mdessi@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

File: hack/OSBS/build-osbs.sh [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+fi
+
+builduser=
+if [ -n "$OSBS_BUILD_USER" ]; then
+    builduser="$OSBS_BUILD_USER"
+fi
+
+cd ../
+# Building the operator
+# TODO: WHICH target ?
```
Comment (spolti) [inline hack/OSBS/build-osbs.sh:207]: @rgdoliveira might know.

Comment (rgdoliveira) [reply hack/OSBS/build-osbs.sh:207]: @spolti sorry, I didn't understand the question properly. What target are you asking about?

Comment (desmax74) [reply hack/OSBS/build-osbs.sh]: Fixed

File: hack/image-osbs.patch [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+-            - ppc64le
+-
+ - name: "rhpam-7/rhpam-kogito-operator"
+   version: "7.11.0"
+   from: "registry.access.redhat.com/ubi8/ubi-minimal:latest"
+@@ -48,14 +39,6 @@
+   packages:
+     content_sets_file: content_sets.yaml
+ 
+-  osbs:
```
Comment (radtriste) [inline hack/image-osbs.patch:25]: are you sure about this removal ? where would be the osbs field then ?

File: hack/image-osbs-bundle.patch [hunk 3]
```diff
@@ -0,0 +1,23 @@
+diff --git a/rhpam-image-bundle.yaml b/rhpam-image-bundle.yaml
+index 094012ee..952e0736 100644
+--- a/rhpam-image-bundle.yaml
++++ b/rhpam-image-bundle.yaml
+@@ -51,18 +51,6 @@ modules:
+ packages:
+   content_sets_file: content_sets.yaml
+ 
+-osbs:
```
Comment (radtriste) [inline hack/image-osbs-bundle.patch:9]: are you sure about the removal of `osbs`  field ?

**Comment by desmax74** [comment]
Replace by https://github.com/kiegroup/kogito-operator/pull/1364

**Pull request closed without merging**